docs/changelog.rst

```diff
@@ -142,6 +142,7 @@ New features:
 
 Bug fixes:
 
+* Fix retry-logic for Spotify plugin.
```

Comment:
It's nice to prefix plugin-specific fixes with `:doc:`/plugins/spotify`. Also maybe this could be slightly more specific: "Add a limit of 3 retries, instead of retrying endlessly when the API is not available."